src/content/wiki/addons.mdx

```diff
@@ -92,6 +92,12 @@ The addon manager is a [webview](https://code.visualstudio.com/api/extension-gui
 
 If you have feedback on the addon manager, please leave it in the [vscode-lua repository](https://github.com/LuaLS/vscode-lua) <Icon name="heart" group="solid" color="red"/>.
 
+To publish your own addon to addon manager,you can refer this [Github Disscussion #2408](https://github.com/LuaLS/lua-language-server/discussions/2408)
+
+<Remark type="note">
+  Because the addons manager's online addons service is based on the git repository on Github, if you can't use the online addons service (Github fetch failure), check your network has access to Github.
```

Comment:
```suggestion
  The addon manager uses Git to retrieve addons, make sure you have access to GitHub if you receive failures to fetch.
```

Reply:
okay,i commited it
now?

Reply:
I still think the remark could be reworded to be a bit shorter while still letting users know they need access to Github.